src/OrchardCore.Modules/OrchardCore.Tenants/ViewModels/AdminIndexViewModel.cs

```diff
@@ -59,7 +69,7 @@ public class TenantIndexOptions
         public List<SelectListItem> TenantsBulkAction { get; set; }
     }
 
-    public enum TenantsFilter
+    public enum TenantsStatus
```

Comment:
If we're going to rename this is should be `TenantState`  or `TenantsState` not status.

Otherwise looks good. 
Can you provide a new graphic please.

Reply:
> Can you provide a new graphic please.

For? I already attached the latest screenshoort [here](https://github.com/OrchardCMS/OrchardCore/pull/10586#issuecomment-991077437)

Reply:
Because the last screenshot didn't make sense, and I had to comment on it requesting more changes.

So I can see what you have done this time, and make sure it makes sense, please include a new screenshot

Reply:
Sorry, seems I removed the `None` badge as you suggested earlier. I need to update the screenshot